src/brsTypes/components/RoSGNode.ts

```diff
@@ -565,15 +631,7 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
             returns: ValueKind.Boolean,
         },
         impl: (interpreter: Interpreter, child: BrsType) => {
-            if (child instanceof RoSGNode) {
-                if (this.children.includes(child)) {
-                    return BrsBoolean.True;
-                }
-                this.children.push(child);
-                child.setParent(this);
-                return BrsBoolean.True;
-            }
-            return BrsBoolean.False;
+            return this.appendChildToParent(child) ? BrsBoolean.True : BrsBoolean.False;
```

Comment:
Nit: 
```suggestion
            return BrsBoolean.from(this.appendChildToParent(child));
```

Reply:
Oh oops I thought I changed all of that. Will do